jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/PrivateMessageControllerTest.java

```diff
@@ -365,52 +366,24 @@ public void testSaveDraftGet() throws Exception {
     }
 
     @Test
-    public void saveDraftShouldRedirectToDraftsIfValidationErrorsOccurs() throws Exception {
-        String[] errorFields = new String[] {"title", "body", "recipient"};
-        Validator validator = new ValidatorStub(errorFields);
-        mockMvc = MockMvcBuilders.standaloneSetup(controller).setValidator(validator).build();
-
-        mockMvc.perform(post("/pm/save")
-                .param("title", TITLE)
-                .param("body", BODY)
-                .param("recipient", USERNAME))
-                .andExpect(status().isMovedTemporarily())
-                .andExpect(redirectedUrl("/drafts"));
+    public void saveDraftShouldBackToPmFormIfValidationErrorsOccurs() throws Exception {
+        mockMvc = MockMvcBuilders.standaloneSetup(controller).build();
```

Comment:
You probably don't want to initialize these for every test.. mockMvc object can be shared.